server/lib/tuist/xcode/xcode_target.ex

```diff
@@ -24,6 +24,28 @@ defmodule Tuist.Xcode.XcodeTarget do
     field :selective_testing_hit, Ch, type: "Enum8('miss' = 0, 'local' = 1, 'remote' = 2)"
     field :xcode_project_id, Ch, type: "UUID"
     field :command_event_id, Ch, type: "UUID"
+    field :product, Ch, type: "LowCardinality(String)", default: ""
+    field :bundle_id, Ch, type: "String", default: ""
+    field :product_name, Ch, type: "String", default: ""
+
+    # Subhashes
+    field :sources_hash, Ch, type: "String", default: ""
```

Comment:
Have you considered adopting the convention `hash_` such that it makes it obvious as you start reading the attribute name that it represents a hash?

Reply:
The current convention for other fields related to hash is with `_hash`, so I'd keep that honestly. I personally don't see much of a difference